templates/db_open_yml.go

```diff
@@ -0,0 +1,15 @@
+package templates
+
+const DB_OPEN_YML = `
+---
+- name: 'Trellis CLI: Open database connection'
+  hosts: web:&{{ env }}
+  remote_user: "{{ web_user }}"
+  gather_facts: false
+  connection: local
+  tasks:
+    - name: Open database connection
+      shell: "open 'mysql+ssh://{{ web_user }}@{{ ansible_host }}:{{ ansible_port | default('22') }}/{{ site_env.db_user }}:{{ site_env.db_password }}@{{ site_env.db_host }}/{{ site_env.db_name }}?enviroment={{ site_env.wp_env }}&usePrivateKey=true'"
```

Comment:
🤔 Wonder if there's an easy to have this playbook just dump out the variables we need as JSON/YAML. Then trellis-cli could parse it and handle actually executing the shell command. I think it's a bit better to keep these "embedded" playbooks as dumb as possible. What do you think?

Reply:
It is possible. Similar to the `dotenv` command.

I guess we are going to dump them in `$TMP` (or `TMP_DIR`).
Do we need to worry about they are not being deleted if `$ trellis db open` half way through?
(note the dump contains db password)

Reply:
Oh I was just assuming we'd get the output from STDOUT and parse it so there wouldn't be a need for any temp files.

Reply:
I worry getting from STDOUT would be troublesome.
Ansible output is not easy to parse and ansible updates might break the parser